cou/steps/plan.py

```diff
@@ -387,17 +387,80 @@ def _get_pre_upgrade_steps(analysis_result: Analysis, args: CLIargs) -> list[Pre
             )
         )
 
+    steps.extend(_get_pre_clean_data_steps(analysis_result, args))
```

Comment:
This change was my suggestion :grimacing:. I agree that for `_get_archive_data_steps` the verbosity increased without need, but for `_get_purge_data_steps` I see some benefit because in the end of the day we just want to add the purge and the details are not necessary while reading `_get_pre_upgrade_steps`.

Reply:
I refactor this part on the last commit.